ports/rp2/boards/WEACTSTUDIO_RP2350B_CORE/modules/board.py

```diff
@@ -0,0 +1,10 @@
+import micropython
+import deflate
+import io
+import binascii
```

Comment:
I think `micropython` and `binascii` can be removed.

Reply:
Done